midterm-2023-11-06/code/5175-kenan-sipahi/index.html

```diff
@@ -9,14 +9,33 @@
 </head>
 <body>
     <h1>Bands Information</h1>
+    <div id="filter" class="filter">
+        
+      <label for="name">Search name: </label>
```

Comment:
The `for` in the label should refer to an `id` attribute of the input, so in this case, the correct value is `name-filter`